static/app/components/feedback/list/feedbackListItem.tsx

```diff
@@ -37,22 +37,6 @@ interface Props {
   style?: CSSProperties;
 }
 
-export function FeedbackIcon({
-  tooltipText,
-  icon,
-}: {
-  icon: ReactNode;
-  tooltipText: string;
-}) {
-  return (
-    <StyledTooltip
-      title={<span style={{textTransform: 'capitalize'}}>{tooltipText}</span>}
```

Comment:
With the recent addition of `getPluginNames`  and the other `get*Names` functions inside issueTrackingSignals, turns out we don't need the textTransform anymore.